onetile.py

```diff
@@ -200,18 +200,22 @@ def valid_output_path(val: str) -> str:
         assert os.path.isdir(val), repr(val)
         return val
 
-    parser = argparse.ArgumentParser()
+    parser = argparse.ArgumentParser(description=__doc__,
```

Comment:
Put arguments on new line, as below.  Switch to that style in launchmany too please 😄